src/dcnum/write/queue_collector_base.py

```diff
@@ -1,71 +1,57 @@
-import logging
 from collections import deque
-import queue
+from typing import List
+import time
 import multiprocessing as mp
+import queue
 import threading
-import time
-from typing import List
-
 import numpy as np
+import logging
+
+mp_spawn = mp.get_context("spawn")
 
 
 class EventStash:
     def __init__(self,
                  index_offset: int,
                  feat_nevents: List[int]):
-        """Sort events into predefined arrays for bulk access
+        """Sorts events into predefined arrays for bulk access.
```

Comment:
The original doc-string was better. Always use imperative language. Don't append a dot. @Orange99